diff --git a/dist/util/action.js b/dist/util/action.js
index c36b7db32346f787c37b8ca6f10e3f55a983cd01..210d5289ba83bd6bde172a38a99679d9a9624668 100644
--- a/dist/util/action.js
+++ b/dist/util/action.js
@@ -1,38 +1,38 @@
-'use strict';
+"use strict";
 
 module.exports = (key, isSelect) => {
-  if (key.meta && key.name !== 'escape') return;
+  if (key.meta && key.name !== "escape") return;
 
   if (key.ctrl) {
-    if (key.name === 'a') return 'first';
-    if (key.name === 'c') return 'abort';
-    if (key.name === 'd') return 'abort';
-    if (key.name === 'e') return 'last';
-    if (key.name === 'g') return 'reset';
+    if (key.name === "a") return "first";
+    if (key.name === "c") return "abort";
+    if (key.name === "d") return "abort";
+    if (key.name === "e") return "last";
+    if (key.name === "g") return "reset";
   }
 
   if (isSelect) {
-    if (key.name === 'j') return 'down';
-    if (key.name === 'k') return 'up';
+    if (key.name === "j") return "down";
+    if (key.name === "k") return "up";
   }
 
-  if (key.name === 'return') return 'submit';
-  if (key.name === 'enter') return 'submit'; // ctrl + J
+  if (key.name === "return") return "submit";
+  if (key.name === "enter") return "submit"; // ctrl + J
 
-  if (key.name === 'backspace') return 'delete';
-  if (key.name === 'delete') return 'deleteForward';
-  if (key.name === 'abort') return 'abort';
-  if (key.name === 'escape') return 'exit';
-  if (key.name === 'tab') return 'next';
-  if (key.name === 'pagedown') return 'nextPage';
-  if (key.name === 'pageup') return 'prevPage'; // TODO create home() in prompt types (e.g. TextPrompt)
+  if (key.name === "backspace") return "delete";
+  if (key.name === "delete") return "deleteForward";
+  if (key.name === "abort") return "abort";
+  if (key.name === "escape") return "abort";
+  if (key.name === "tab") return "next";
+  if (key.name === "pagedown") return "nextPage";
+  if (key.name === "pageup") return "prevPage"; // TODO create home() in prompt types (e.g. TextPrompt)
 
-  if (key.name === 'home') return 'home'; // TODO create end() in prompt types (e.g. TextPrompt)
+  if (key.name === "home") return "home"; // TODO create end() in prompt types (e.g. TextPrompt)
 
-  if (key.name === 'end') return 'end';
-  if (key.name === 'up') return 'up';
-  if (key.name === 'down') return 'down';
-  if (key.name === 'right') return 'right';
-  if (key.name === 'left') return 'left';
+  if (key.name === "end") return "end";
+  if (key.name === "up") return "up";
+  if (key.name === "down") return "down";
+  if (key.name === "right") return "right";
+  if (key.name === "left") return "left";
   return false;
 };
diff --git a/lib/util/action.js b/lib/util/action.js
index fefbd947a1307620330c3a1ac19e15cbff289c7e..df749530abb05f35b6c706328c856e166dcd7a20 100644
--- a/lib/util/action.js
+++ b/lib/util/action.js
@@ -1,39 +1,39 @@
-'use strict';
+"use strict";
 
 module.exports = (key, isSelect) => {
-  if (key.meta && key.name !== 'escape') return;
-  
+  if (key.meta && key.name !== "escape") return;
+
   if (key.ctrl) {
-    if (key.name === 'a') return 'first';
-    if (key.name === 'c') return 'abort';
-    if (key.name === 'd') return 'abort';
-    if (key.name === 'e') return 'last';
-    if (key.name === 'g') return 'reset';
+    if (key.name === "a") return "first";
+    if (key.name === "c") return "abort";
+    if (key.name === "d") return "abort";
+    if (key.name === "e") return "last";
+    if (key.name === "g") return "reset";
   }
-  
+
   if (isSelect) {
-    if (key.name === 'j') return 'down';
-    if (key.name === 'k') return 'up';
+    if (key.name === "j") return "down";
+    if (key.name === "k") return "up";
   }
 
-  if (key.name === 'return') return 'submit';
-  if (key.name === 'enter') return 'submit'; // ctrl + J
-  if (key.name === 'backspace') return 'delete';
-  if (key.name === 'delete') return 'deleteForward';
-  if (key.name === 'abort') return 'abort';
-  if (key.name === 'escape') return 'exit';
-  if (key.name === 'tab') return 'next';
-  if (key.name === 'pagedown') return 'nextPage';
-  if (key.name === 'pageup') return 'prevPage';
+  if (key.name === "return") return "submit";
+  if (key.name === "enter") return "submit"; // ctrl + J
+  if (key.name === "backspace") return "delete";
+  if (key.name === "delete") return "deleteForward";
+  if (key.name === "abort") return "abort";
+  if (key.name === "escape") return "abort";
+  if (key.name === "tab") return "next";
+  if (key.name === "pagedown") return "nextPage";
+  if (key.name === "pageup") return "prevPage";
   // TODO create home() in prompt types (e.g. TextPrompt)
-  if (key.name === 'home') return 'home';
+  if (key.name === "home") return "home";
   // TODO create end() in prompt types (e.g. TextPrompt)
-  if (key.name === 'end') return 'end';
+  if (key.name === "end") return "end";
 
-  if (key.name === 'up') return 'up';
-  if (key.name === 'down') return 'down';
-  if (key.name === 'right') return 'right';
-  if (key.name === 'left') return 'left';
+  if (key.name === "up") return "up";
+  if (key.name === "down") return "down";
+  if (key.name === "right") return "right";
+  if (key.name === "left") return "left";
 
   return false;
 };
