diff --git a/lib/util/action.js b/lib/util/action.js
index fefbd947a1307620330c3a1ac19e15cbff289c7e..fcb1df8e74f1e2758692dbe4fdc8f3323986cd4c 100644
--- a/lib/util/action.js
+++ b/lib/util/action.js
@@ -21,7 +21,7 @@ module.exports = (key, isSelect) => {
   if (key.name === 'backspace') return 'delete';
   if (key.name === 'delete') return 'deleteForward';
   if (key.name === 'abort') return 'abort';
-  if (key.name === 'escape') return 'exit';
+  if (key.name === 'escape') return 'abort';
   if (key.name === 'tab') return 'next';
   if (key.name === 'pagedown') return 'nextPage';
   if (key.name === 'pageup') return 'prevPage';
